bundles/org.openhab.binding.tacmi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -163,7 +163,7 @@
 		<item-type>Number</item-type>
 		<label>Value</label>
 		<description>A numeric value read from C.M.I.</description>
-		<state readOnly="true"/>
+		<state readOnly="true" pattern="%.1f %unit%"/>
```

Comment:
Your channel is mentioned to be linkable to a simple Number item type so please do not use UoM, that is do not use `%unit%`.

Reply:
Seems fixed